src/libexpr/eval.hh

```diff
@@ -31,6 +31,7 @@ struct PrimOp
     Symbol name;
     std::vector<std::string> args;
     const char * doc = nullptr;
+    std::optional<std::string> missingFeature;
```

Comment:
```suggestion
    std::optional<std::string> requiredFeature;
```

(Unless you really want it to mean `missingFeature`, in which case I assume it should only be non-null if the feature is actually missing, but that’s not what the code is doing currently)